tools/doc/gpcheck_help

```diff
@@ -135,27 +142,27 @@ OPTIONS
 EXAMPLES
 *****************************************************
 
-Verify and validate the HAWQ platform settings by 
-entering a host file and specifying the hadoop location:
+Verify and validate the HAWQ platform settings by  specifying
+a host file and identifying the full hadoop install path:
 
- # hawq check -f hostfile_hawq_check --hadoop ~/hadoop-2.0.0/
+ # hawq check -f hostfile_hawq_check --hadoop /usr/hdp/<version>/hadoop/
```

Comment:
We have both '/usr/hdp/<version>/hadoop/' and '/usr/local/hadoop-<version>/'. Maybe we should make them the same.